packages/core/lib/src/core_data.dart

```diff
@@ -84,7 +84,7 @@ class BuildOp {
   /// The execution priority, op with lower priority will run first.
   ///
   /// Default: 10.
-  final int priority;
+  final int? priority;
```

Comment:
`priority` has a default value of `10` and it is used to sort the ops so it should not be null.